flashlib.core.robot/src/main/java/com/flash3388/flashlib/robot/control/PidController.java

```diff
@@ -60,15 +59,16 @@ public PidController(DoubleProperty kp, DoubleProperty ki, DoubleProperty kd, Do
         mOutRampRate = 0;
 
         mTotalError = 0;
+        mLastError = 0;
+        mLastTimeSeconds = Time.seconds(0);
```

Comment:
You can use `Time.INVALID` instead, since you replace it anyhow.

Reply:
nevermind